src/ONYXKEYS.ts

```diff
@@ -1111,6 +1120,9 @@ type OnyxValuesMapping = {
     [ONYXKEYS.CORPAY_ONBOARDING_FIELDS]: OnyxTypes.CorpayOnboardingFields;
     [ONYXKEYS.LAST_FULL_RECONNECT_TIME]: string;
     [ONYXKEYS.TRAVEL_PROVISIONING]: OnyxTypes.TravelProvisioning;
+    [ONYXKEYS.IS_LOADING_BILL_WHEN_DOWNGRADE]: boolean;
+    [ONYXKEYS.SHOULD_BILL_WHEN_DOWNGRADING]: boolean;
```

Comment:
I think the default value for this key is undefined, so its type would be `boolean | undefined`.